docs/guide/advanced/navigation-failures.md

```diff
@@ -0,0 +1,60 @@
+# Navigation Failures
+
+> New in 3.4.0
+
+When using `router-link`, Vue Router internally calls `router.push` to trigger a navigation. Depending on the current location and existing [Navigation Guards](./navigation-guards.md), this navigation might end up in a new page being shown, but there are a couple of situations where we will stay on the same page:
+
+- We are already on the page we are trying to go to
```

Comment:
```suggestion
- Users are already on the page that is being requested
```

Reply:
Should we say _that is being navigated to_? as there isn't any request going on or is it still clearer that way?

Reply:
Updated!